banyan/tests/test_sessions.py

```diff
@@ -0,0 +1,5 @@
+from banyan import start_session
+
+# TODO: Check that the required environment variables are set before
+# running tests: api_key, user_id, endpoint
+
```

Comment:
Can you at least add some documentation here of how you tested this manually?

Reply:
Yes, done.